bsp4ts/src/pkg/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
why such weird formatting?

Reply:
ahh, I see, probably you don't strip the margin somewhere in the generator

Reply:
Yeah looks like I was missing a trimIndent